src/services/auth.service.ts

```diff
@@ -1,41 +1,142 @@
 import bcrypt from 'bcrypt';
-import jwt from "jsonwebtoken";
-import {AppDataSource} from "../config/data-source"
-import {User} from "../entities/User.entity"
+import jwt from 'jsonwebtoken';
+import { AppDataSource } from '../config/data-source';
+import { User } from '../entities/User.entity';
+import { Role } from '../entities/Role.entity';
+import { HttpException } from '../exceptions/HttpException';
+// import { HttpException } from '../middlwares/errorHandler';
+import { MailService } from './mail.service';
+import 'dotenv/config';
 
-const ACCESS_SECRET = "access_secret"
-const REFRESh_SECRET =  "refresh_secret"
+const VERIFY_SECRET = process.env.VERIFY_SECRET!;
+const ACCESS_SECRET = process.env.ACCESS_SECRET!;
+const REFRESH_SECRET = process.env.REFRESH_SECRET!;
 
-export class AuthService{
-    private userRepo = AppDataSource.getRepository(User);
+console.log('ACCESS_SECRET in auth.service:', process.env.ACCESS_SECRET);
```

Comment:
Remove logs to enhance performance.
Create a repository to handle database operations.